storybook/src/docs/form/components/NumberField.stories.mdx

```diff
@@ -0,0 +1,27 @@
+import { Meta, Canvas, Story, Source } from "@storybook/addon-docs";
+
+<Meta title="Docs/Form/Components/NumberField" />
+
+# Number Field
+
+The NumberField takes a number and formats it according to local standard formatting.
```

Comment:
The primary job of this field is not to format a number, but to be able to input a number (with thousand-separator and decimal symbol, in the current locale) while storing it as type `number` in the form values.